hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -106,7 +106,7 @@ private FlinkOptions() {
           + "determined by Object.compareTo(..)");
 
   public static final ConfigOption<String> PAYLOAD_CLASS_NAME = ConfigOptions
-      .key("payload.class")
```

Comment:
For a public config option, we should not remove it directly. It would cause a compatible issue.

Reply:
Not sure why remove this, we already have a fallback key: `write.payload.class`

Reply:
@danny0405  Because the user looked at the documentation and thought that write.payload.class could not take effect, I thought it might be necessary to change the name of the configuration item, has now been updated